library/src/main/java/com/novoda/downloadmanager/MigrationServiceBinder.java

```diff
@@ -19,19 +19,22 @@
     }
 
     public void bind() {
-        if (serviceConnection == null) {
-            serviceConnection = new MigrationServiceConnection();
+        if (serviceConnection != null) {
```

Comment:
We are assuming that this is only being executed on an `Application` level and therefore will only be allowed to be bound once.